snippets/chain-async-functions.md

```diff
@@ -0,0 +1,40 @@
+### Chain asynchronous functions
+
+Loop through an array of functions containing asynchronous events, calling `next` when each asynchronous event has completed.
+
+```js
+const chainAsync = fns => {
+	let curr = 0; const next = () => fns[curr++](next); next()
+}
```

Comment:
Provide an example in a comment, below the declaration, please.

Reply:
@Chalarangelo Done.